node/src/tasks/mock_bsp_volunteer.rs

```diff
@@ -5,49 +5,49 @@ use shc_actors_framework::event_bus::EventHandler;
 use shc_blockchain_service::{commands::BlockchainServiceInterface, events::NewStorageRequest};
 use shc_common::types::StorageProofsMerkleTrieLayout;
 use shc_file_manager::traits::FileStorage;
-use shc_forest_manager::traits::ForestStorage;
+use shc_forest_manager::traits::ForestStorageHandler;
 use sp_core::H256;
 
 use crate::services::handler::StorageHubHandler;
 
 const LOG_TARGET: &str = "bsp-volunteer-mock-task";
 
-pub struct BspVolunteerMockTask<FL, FS>
+pub struct BspVolunteerMockTask<FL, FSH>
 where
-    FL: Send + Sync + FileStorage<StorageProofsMerkleTrieLayout>,
-    FS: Send + Sync + ForestStorage<StorageProofsMerkleTrieLayout> + 'static,
+    FL: FileStorage<StorageProofsMerkleTrieLayout> + Send + Sync,
+    FSH: ForestStorageHandler + Clone + Send + Sync + 'static,
 {
-    storage_hub_handler: StorageHubHandler<FL, FS>,
+    storage_hub_handler: StorageHubHandler<FL, FSH>,
 }
 
-impl<FL, FS> Clone for BspVolunteerMockTask<FL, FS>
+impl<FL, FSH> Clone for BspVolunteerMockTask<FL, FSH>
 where
-    FL: Send + Sync + FileStorage<StorageProofsMerkleTrieLayout>,
-    FS: Send + Sync + ForestStorage<StorageProofsMerkleTrieLayout> + 'static,
+    FL: FileStorage<StorageProofsMerkleTrieLayout> + Send + Sync,
```

Comment:
I think it's worth moving the `Send`, `Sync`, `'static' and `Clone` bounds in the `FileStorage` so that you don't have to specify it everywhere. What do you think?